src/chrome/content/rules/Al-Madina.com.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Mismatched:
+		madina.al-madina.com, CN=*.prod.ams3.secureserver.net.
+	Secure Connection Failed:
+		waseet.al-madina.com
```

Comment:
Same with this one.

Reply:
strange got a secure connection failed yesterday :thinking: